python/tank/util/errors.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from ..errors import TankError
+
+class UnresolvableCoreConfigurationError(TankError):
```

Comment:
To follow the established core patterns, we should import this in `sgtk.util.__init__` and then when the exception is being imported from a location outside of `sgtk.util` it should be done via a `from ...util import UnresolvableCoreConfigurationError`.